packages/aikit-ui/src/lib/openai-transport.ts

```diff
@@ -66,6 +66,7 @@ export class OpenAIHttpChatTransport<
           : undefined;
 
         return {
+          credentials: "include",
```

Comment:
For security reasons, this should not be the default, but instead be injected via external fetch instance.